src/webview/MarkdownEngine.ts

```diff
@@ -0,0 +1,109 @@
+import * as hljs from "highlight.js";
+import * as MarkdownIt from "markdown-it";
+import * as os from "os";
+import * as path from "path";
+import * as vscode from "vscode";
+import { leetCodeChannel } from "../leetCodeChannel";
+
+export class MarkdownEngine {
+
+    public readonly engine: MarkdownIt;
```

Comment:
Can be private now...

Reply:
I forgot to change it when resolving the conversation...